BrainPortal/lib/exception_helpers.rb

```diff
@@ -38,36 +41,51 @@ def self.included(includer) #:nodoc:
 
   # Record not accessible.
   def record_not_found(exception)
-    raise unless Rails.env == 'production' #Want to see stack trace in dev.
-    flash[:error] = "The object you requested does not exist or is not accessible to you."
+    raise if Rails.env == 'development' #Want to see stack trace in dev.
```

Comment:
At two places you changed the exception forwarding conditional from

```ruby
raise unless Rails.env == 'production'
```

to

```ruby
raise if Rails.env == 'development'
```

The difference in behavior is only while in the `test` environment. Is there a good reason to have this change? This file contains two other methods with similar checks and they have been left with the original check (`== 'production'`). I don't like that we have two methods with one way, and two methods the other way. Can you double-check that please? It's particularly weird because the original code is 7 years old (from the original file commit made by @natacha-beck in April 2012) so I don't see why it would suddenly no longer be appropriate.